pyproject.toml

```diff
@@ -17,7 +17,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.13"
 pyquil = "^4.14.3"
-azure-quantum = ">=0.27,<1.0"
+azure-quantum = ">=1.0,<2.0"
```

Comment:
Since no actual code changes are necessary in this repo to make the tests pass, we _could_ simply specify `azure-quantum = "<3.0"`, so that it doesn't matter which version a user wants to use. (Though if we do that, we should probably add it to the already-broad testing matrix.)

Reply:
I've changed the constraint to `<3.0`.